app/components/utilityBookingForm.tsx

```diff
@@ -60,8 +62,15 @@ function UtilityBookingForm(props: UtilityBookingFormProps) {
           </div>
         </div>
         <br />
+        {props.validBooking == 1 && <p> Booking successful! </p>}
+        {props.validBooking == 1 && <p> From: {props.bookingFrom} </p>}
+        {props.validBooking == 1 && <p> To: {props.bookingTo} </p>}
+
+        {props.validBooking === 0 && <p>Booking unsuccessful!</p>}
+        {props.validBooking === 0 && <p>Possible reasons:</p>}
+        {props.validBooking === 0 && <p>- *To* needs to be after *From*</p>}
+        {props.validBooking === 0 && <p>- Utility is already booked</p>}
```

Comment:
You can group the props.validBooking bool calls together like
```
{props.validBooking == 1 && (
    <>
        <p> Booking successful! </p>
        <p> From: {props.bookingFrom} </p>
        <p> To: {props.bookingTo} </p>
    </>
)}
```
Same goes for when validBooking is 0!